nixos/modules/programs/weylus.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.weylus;
+in
+{
+  options.programs.weylus = with types; {
+    enable = mkEnableOption "weylus";
+
+    openPorts = mkOption {
+      type = bool;
+      default = false;
+      description = ''
+        Open ports needed for the functionality of the program.
+      '';
+    };
+
+     users = mkOption {
+      type = listOf str;
+      default = [ ];
+      description = ''
+        Weylus uses the uinput interface to simulate input events on Linux. To enable stylus and multi-touch support the user you're going to use must be added to this list. This allows them to synthesize input events system-wide, even when another user is logged in. Therefore, untrusted users should not be added to this list.
```

Comment:
Well... maybe line split this a bit too.

Reply:
Done, cleaned it up a bit as well. It was copied it straight from the package's readme